### PR TITLE
Change curl image

### PR DESCRIPTION
**Pull request by ivyostosh**
*Issue #, if available:*

*Description of changes:* Change the dockerhub image to an EKSA built image.

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pkg/executables/kubectl.go [hunk 1]
```diff
@@ -991,7 +991,7 @@ func (k *Kubectl) ValidatePods(ctx context.Context, kubeconfig string) error {
 
 // RunBusyBoxPod will run Kubectl run with a busybox curl image and the command you pass in.
 func (k *Kubectl) RunBusyBoxPod(ctx context.Context, namespace, name, kubeconfig string, command []string) (string, error) {
```
Comment (raymond-zhang00) [inline pkg/executables/kubectl.go:993]: Could you rename this func

Comment (ivyostosh) [reply pkg/executables/kubectl.go]: updated

**Pull request merged**